contracts/0.8.9/Burner.sol

```diff
@@ -135,7 +130,22 @@ contract Burner is IBurner, AccessControlEnumerable {
 
         LOCATOR = ILidoLocator(_locator);
         LIDO = ILido(_stETH);
+    }
+
+    /**
+     * @param _totalCoverSharesBurnt Shares burnt counter init value (cover case)
+     * @param _totalNonCoverSharesBurnt Shares burnt counter init value (non-cover case)
+     * @dev Although there is initialize function, the contract is not supposed to be upgradeable.
+     *      The initialization is moved from the constructor to be able to set the shares burnt counters
+     *      to the up-to-date values from the old Burner upon the upgrade.
+     */
+    function initialize(
```

Comment:
:warning: 
- we have to migrate the whole state, the most important is about `*BurnRequested`
- transfer stETH back here :warning:

Reply:
let's pass the prev burner address

Reply:
:brain: should be a part of the finalizeUpgrade_Vx for `Lido`